.github/workflows/build.yml

```diff
@@ -1,11 +1,6 @@
 name: Build
-on:
```

Comment:
I figured code might as well be tested and linted on any push regardless of branch. Let me know if you disagree.